CMakeLists.txt

```diff
@@ -15,6 +15,10 @@ if(NOT OpenVINODeveloperPackage_FOUND)
     find_package(OpenVINO REQUIRED COMPONENTS Runtime)
 endif()
 
+set_target_properties(openvino::runtime PROPERTIES
+  MAP_IMPORTED_CONFIG_MINSIZEREL Release
+  MAP_IMPORTED_CONFIG_RELWITHDEBINFO Release)
```

Comment:
https://github.com/openvinotoolkit/openvino/blob/master/cmake/templates/OpenVINOConfig.cmake.in#L508

If something needs to be fixed => the proper place is `OpenVINOConfig.cmake`